cfpb/consumerfinance.gov
# Hide archive filter options in the filterable UI

CE wants the archive filter options in filterable lists to be hidden.
They are placed behind a feature flag so that it's possible to enable and disable if we need to.

## Additions

- `HIDE_ARCHIVE_FILTER_OPTIONS` feature flag

## Changes

- Modified `filterable_list_mixins` model to use `HIDE_ARCHIVE_FILTER_OPTIONS` flag

## How to test this PR

1. Enable `HIDE_ARCHIVE_FILTER_OPTIONS` flag in Wagtail admin
2. Navigate to `/about-us/blog/` page and look at filter posts
3. Click on Show filters on `/data-research/research-reports/` page and look at filter reports
4. Navigate to `/about-us/newsroom/` and look at filter posts
5. Verify archive options are removed from filterable UI

## Screenshots

## Notes and todos

- demo this feature to CE before it goes live

## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance

### Front-end testing

#### Browser testing

Visually tested in the following supported browsers:

#### Accessibility


#### Other
